README.md

```diff
@@ -51,3 +72,149 @@ using the following command:
 ```shell
 $ ENABLE_WEBHOOKS=false make run install
 ```
+
+## Testing the operator using Kind
+
+The following steps were tested on Fedora 38 as it runs seamlessly but it may be reproduced in any Linux distribution.
+
+For RHEL users: Extra steps might be required ([Kind - Rootless](https://kind.sigs.k8s.io/docs/user/rootless/)) but as of May 2023 there was no success running Kind after applying them.
+
+### Installing `kind`
+```
+$ go install sigs.k8s.io/kind@v0.19.0
+``` 
+
+### Installing `kubectl`
+
+The following link shows the steps to install kubectl: https://kubernetes.io/docs/tasks/tools/
+
+### Creating the clusters
+
+Create the `local` and `remote` clusters and install tekton latest on them.
+Use two different terminals for that - one for each cluster.
+
+#### `local` cluster terminal
+```
+$ mkdir ~/.kube
+$ export KUBECONFIG=~/.kube/local
+$ kind create cluster --name=local
+$ kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+```
+#### `remote` cluster terminal
+```
+$ export KUBECONFIG=~/.kube/remote
+$ kind create cluster --name=remote
+$ kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+```
+### Clone and run the operator
+```
+$ git clone git@github.com:redhat-appstudio/internal-services.git
+Cloning into 'internal-services'...
+```
+#### Installing the internal-services CR
+
+Both clusters should have the CRs installed, so the following steps should be done in both terminals, local and remote.
+```
+$ cd internal-services
+internal-services $ make manifests generate
+internal-services $ make install
+```
+
+#### Running the operator in the cluster
+
+The operator needs to run a single cluster - in this test case, the `local` cluster/terminal - and it
+will listen for `InternalRequests` created in the `remote` cluster denoted by the `--remote-cluster-config-file` parameter. 
+
+```
+internal-services $ go run main.go --remote-cluster-config-file=$HOME/.kube/remote
+```
+
+### Testing an InternalRequest
+
+To process a `InternalRequest` the `internal-services-controller` requires a Pipeline created in the Cluster that the request should run in.
+
```

Comment:
Need to apply `internalServiceConfig.yaml` in the `remote` cluster
Sample Config
```
cat internal-config.yaml 
apiVersion: appstudio.redhat.com/v1alpha1
kind: InternalServicesConfig
metadata:
  name: "config"
spec:
  allowList:
    - default
  debug: true
  volumeClaim:
    name: workspace
    size: 1Gi
```
`$ kubectl create -f internal-config.yaml`

Also, add a Note Internal Request should be created in the namespace mentioned in the `allowList:` of internalServiceConfig.